src/DocumentFormat.OpenXml/SimpleTypes/EnumValue.cs

```diff
@@ -10,15 +10,14 @@ namespace DocumentFormat.OpenXml
     /// Represents the enum value for attributes.
     /// </summary>
     /// <typeparam name="T">Every enum value must be an enum with the EnumStringValueAttribute object.</typeparam>
-    [DebuggerDisplay("{InnerText}")]
+    [DebuggerDisplay("{" + nameof(InnerText) + "}")]
     public class EnumValue<T> : OpenXmlSimpleValue<T>
         where T : struct
```

Comment:
For v3.0, we can add a constraint for Enum (as of C# 7.3) :)

Reply:
@twsouthwick Do you want me to do anything here? ReSharper suggested this change and I've used `nameof` in my code as much as possible to avoid issues as names change (which might not be the case here).

Reply:
Yeah, I saw that change as I quickly glanced through. I like the use of `nameof` as much as possible and see the value here. However, that name won't be changing, so I'd probably not make the change myself as it takes a bit more time to grok. I'll leave that up to you, though